docs/mkdocs-html.yml

```diff
@@ -97,21 +97,21 @@ plugins:
         #'old.md': 'new.md'
         #'old/file.md': 'new/file.md'
         #'some_file.md': 'http://external.url.com/foobar'
-  - enumerate-headings:
-      toc_depth: 3
-      strict: true
-      increment_across_pages: true
-      include:
-        - "*"
-      exclude:
-        - index.md
-        - user/index.md
-        - administrator/index.md
-        - developer/index.md
-        - developer/manual/*.md
-        - devops/index.md
-      restart_increment_after:
-        - second_section.md
+  # - enumerate-headings:
+  #     toc_depth: 3
+  #     strict: true
+  #     increment_across_pages: true
+  #     include:
+  #       - "*"
+  #     exclude:
+  #       - index.md
+  #       - user/index.md
+  #       - administrator/index.md
+  #       - developer/index.md
+  #       - developer/manual/*.md
+  #       - devops/index.md
+  #     restart_increment_after:
+  #       - second_section.md
```

Comment:
We need the enumerate headings plugin for this docs site.

Reply:
Okay, I will uncomment this then.